store/helper/helper.go

```diff
@@ -862,6 +862,17 @@ func requestPDForOneHost(host, apiName, method, uri string, body io.Reader, res
 				zap.String("url", urlVar), zap.Error(err))
 		}
 	}()
+
+	if resp.StatusCode != http.StatusOK {
+		bs, _ := io.ReadAll(resp.Body)
+		logutil.BgLogger().Warn("requestPDForOneHost failed with non 200 status",
+			zap.String("url", urlVar), zap.Error(err),
```

Comment:
Is `err` related to the failure at this point?

Reply:
It's mistake when copy and paste. We should not log the error here because it is not related...